src/deepsparse/transformers/helpers.py

```diff
@@ -44,7 +46,84 @@
 _LOGGER = get_main_logger()
 
 
-def get_deployment_path(model_path: str) -> Tuple[str, str]:
+def setup_transformers_pipeline(
```

Comment:
Could we also get the v1 pipeline to use these? Fear is that we'll have two places to update going forward.

Reply:
Sure, good idea.